File: src/Domain/Entities/Articulo.cs
using Ophelia.Domain.Common;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Ophelia.Domain.Entities
{
    [Table("articulos", Schema = "facturacion")]
    public class Articulo : AuditableEntity
    {
        public int Id { get; set; }
        public string Detalle { get; set; }
        public int UnidadId { get; set; }
        public virtual Unidad Unidad { get; set; }
        public decimal Precio { get; set; }
    }
}

File: Front/src/app/pages/articulo/model/Articulo.ts
import {IArticulo} from './IArticulo';
export class Articulo implements IArticulo
{
    id: number;
    detalle: number;
    unidadId: number;
    unidadDetalle: string;
    precio: number;
    
    constructor(data?)
    {
        
        data = data || {};
        this.id = data.articulo.id || 0;
        this.detalle = data.articulo.detalle || '';
        this.unidadId = data.articulo.unidadId || '';
        this.unidadDetalle = data.articulo.unidadDetalle || '';
        this.precio = data.articulo.precio || 0;
    }
}